File: DesignPatternsDotNet/Facade/Attendance.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Facade
{
    public class Attendance
    {
        private Student student;
        private double attendance = 85.6;
        public Attendance(Student aStudent)
        {
            student = aStudent;
        }
        public void GetStudentAttendance(){
            Console.WriteLine(student.Name + "'s attendance = " + attendance.ToString("#.##") + " percentage");
	}
    }
}

File: DesignPatternsJava/src/main/java/com/facade/Attendance.java
package com.facade;

public class Attendance {
	private Student student;
	private double attendance = 85.6;
	public Attendance(Student aStudent){
		student=aStudent;
	}
	public void getStudentAttendance(){
		System.out.println(student.getName()+"'s attendance = "+attendance+" percentage");
	}
}
